extensions/vscode/src/extension/VsCodeMessenger.ts

```diff
@@ -168,11 +162,28 @@ export class VsCodeMessenger {
     this.onWebview("openInventory", (msg) => {
       vscode.commands.executeCommand("pearai.toggleInventoryHome");
     });
-    this.onWebview("completeWelcome", (msg) => {
+    this.onWebview("pearAIinstallation", (msg) => {
+      const { tools, installExtensions } = msg.data;
+      if (installExtensions) {
+        vscode.commands.executeCommand("pearai.welcome.importUserSettingsFromVSCode");
+      }
```

Comment:
its importing extensions in pearAIinstallation. naming can be improved here. but its not a big deal.

Reply:
Very true